sites/website/src/docs/design-systems/high-contrast.md

```diff
@@ -0,0 +1,180 @@
+---
+id: high-contrast
+title: High Contrast in FAST
+sidebar_label: High Contrast
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/design-systems/high-contrast.md
+---
+
+## Styling components using forced-colors.
+High contrast mode uses the CSS media feature, [`forced-colors`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors). When `forced-colors` is set to `active`, the user agent will apply a limited color palette to the component.
+
+
+**Example:**
+```css
+@media (forced-colors: active) {
+    :host {
+        background: ButtonFace;
+    }
+}
+```
+
+FAST has a [forcedColorsStylesheetBehavior](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/utilities/match-media-stylesheet-behavior.ts) utility function that is used to construct `forced-colors` in the stylesheet. This function is passed to the `withBehavior` function from the `css` tagged template object.
+
+:::note
+The reason for this behavior is to avoid the runtime cost applying forced-color style rules when the UA does not match the forced-colors @media query, FAST exposes a behavior that conditionally adds and removes stylesheets based on the forced-colors @media query. forced-colors stylesheets can then be conditionally applied where necessary.
```

Comment:
```suggestion
The reason for this behavior is to avoid the runtime cost of applying forced-color style rules when the UA does not match the forced-colors @media query. FAST exposes a behavior that conditionally adds and removes stylesheets based on the forced-colors @media query. Forced-colors stylesheets can then be conditionally applied where necessary.
```